src/praisonai/praisonai/cli/commands/schedule.py

```
@@ -121,6 +136,23 @@ def schedule_add_cmd(
                         job.command_timeout = command_timeout
+                    if backend:
+                        job.backend = backend
+                        backend_options = {}
+                        if backend_cwd:
+                            backend_options["cwd"] = backend_cwd
+                        if backend_timeout and backend_timeout > 0:
+                            # Round up so a positive sub-millisecond value stays
+                            # positive instead of collapsing to 0 (which the
+                            # executor would replace with the 300s default).
+                            backend_options["timeout_ms"] = max(
+                                1, int(round(backend_timeout * 1000))
```

Ruff (0.16.1)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)